OniProj/OniCMakeProjs/OniProj/CMakeLists.txt

```diff
@@ -502,18 +504,15 @@ add_executable(Oni WIN32
         ../../OniGameSource/Oni_TextureMaterials.c
         ../../OniGameSource/Oni_TextureMaterials.h
         ../../OniGameSource/Oni_Weapon.c
-        ../../OniGameSource/Oni_Weapon.h
-
-        ../../OniResources/OniResources.rc
-        ../../OniResources/resource.h)
+        ../../OniGameSource/Oni_Weapon.h)
 
-target_precompile_headers(Oni PRIVATE ../../../BungieFrameWork/BFW_Headers/BFW_MasterHeader.h)
-target_compile_definitions(Oni PRIVATE ONI_MAP_FILE="OniDbg.map")
-
-if(MSVC)
-    target_compile_definitions(Oni PRIVATE /FI"../../../BungieFrameWork/BFW_Headers/BFW_MasterHeader.h")
+if(WIN32)
+    add_executable(Oni WIN32
+            ${Source_Files}
```

Comment:
I'd almost missed this but keep in mind these are case-sensitive, so it'll need to be changed to `SOURCE_FILES`.